src/test/java/guitests/UITest.java

```diff
@@ -256,6 +257,20 @@ public void waitUntilNodeDisappears(String selector) {
         }
     }
 
+    public void waitUntilNodeAppears(Matcher<Object> matcher) {
+        while (!findQuiet(matcher).isPresent()) { // no `exists` for matcher, so using find
```

Comment:
This is the new function. I implemented `findQuiet` with `find(Matcher<Object>)`. I didn't use `exist` because there is no `exists(Matcher<Object>)` by testfx.